docs/gathered-data.md

```diff
@@ -979,6 +979,30 @@ gathers the alerts that have been silenced.
   - 4.10+
 
 
+## StorageCluster
+
+Collects `storageclusters.ocs.openshift.io` resources
+from Openshift Data Foundation Stack.
+
+### API Reference:
+https://github.com/red-hat-storage/ocs-operator/blob/main/api/v1/storagecluster_types.go
+
+### Sample data
+docs/insights-archive-sample/config/storage/openshift-storage/storageclusters/ocs-storagecluster.json
```

Comment:
Again this is changing the original format. We can change it, but then it should be changed everywhere,

Reply:
Ohh sorry this is new right? I confused myself with `Location in archive:` ...so it's probably OK and you can ignore me